components/datetime/src/provider/calendar/mod.rs

```diff
@@ -132,6 +140,7 @@ pub mod patterns {
         pub short: GenericPattern<'data>,
     }
 
+    /// 
```

Comment:
Nit: Empty comment

Suggestion: I think this (`PatternPluralsV1`) can be a crate-private type. It need not be exported. Please make it `pub(crate)`.

Or are we already inside a crate-private context?